app/payment-request/index.js

```diff
@@ -0,0 +1,43 @@
+const db = require('../data')
+const paymentRequestSchema = require('./payment-request-schema')
+
+async function savePaymentRequest (paymentRequest) {
```

Comment:
I think we need to handle duplicate payment requests, ie if we get the same payment request again, we don't create a new payment request for it in the database and double pay someone.

We'll need to do this in a transaction, but have to check to see if an existing payment request exists with the same values.  If not, then create.  If yes, then just log duplicate to the console for info for now.

Reply:
agree